test/ibmq/test_experiment.py

```diff
@@ -43,9 +43,17 @@ def setUpClass(cls):
         super().setUpClass()
         try:
             cls.provider = cls._setup_provider()    # pylint: disable=no-value-for-parameter
-            cls.experiments = cls.provider.experiment.experiments()
+            cls.experiments = cls.provider.experiment.experiments(tags=['qiskit-test'], limit=None)
```

Comment:
If every call to `cls.provider.experiment.experiments` in this test module is going to include that tags filter, at least by default, I'd suggest adding a helper function that does that, e.g.:

```python
def list_experiments(self, tags=['qiskit-test'], **kwargs):
    return cls.provider.experiment.experiments(tags=tags, **kwargs)
```

Something like that just so you don't need to copy/paste the same thing all over the place for the tags filtering.

Reply:
[Updated! ](https://github.com/Qiskit/qiskit-ibmq-provider/pull/1002/files#diff-9275794e4268a2c8dd165487dab5c87c3d271f9692458ac71902381b701725b6R787)